doc/Command_Index_internal.tex

```diff
@@ -40,24 +40,27 @@ \section{Gregorio\TeX{} Controls}
   \#2 & string & name of the correct macro to use\\
 \end{argtable}
 
+\macroname{\textbackslash gre@obsolete}{\#1\#2}{gregoriotex-main.tex}
+Macro that handles obsolescence errors.
+
+\begin{argtable}
+  \#1 & string & name of the obsolete macro\\
+  \#2 & string & name of the correct macro to use\\
+\end{argtable}
+
 \macroname{\textbackslash gre@loadgregoriofont}{}{gregoriotex-main.tex}
 Loads the chosen font for the neumes at the correct size.
 
 \macroname{\textbackslash gre@calculate@constantglyphraise}{}{gregoriotex-spaces.tex}
-%\verb=\gre@calculateconstantglyphraise=%tex.tex
 Macro to caluclate \verb=\gre@constantglyphraise=
```

Comment:
I know you didn't put this line in but caluclate is misspelled.  Should be calculate.